### PR TITLE
Update transltions from Weblate

### PR DESCRIPTION
**Pull request by zetok**
This change is [Reviewable](https://reviewable.io/reviews/qtox/qtox/3950)

### CODE REVIEW

**Comment by nurupo** [comment]
@zetok has asked me to :lgtm: it without reviewing it, so here it goes.

---







*Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/qtox/3950#-:-KYWqNcRJreLpVKgGmol:b-154vk2)*

File: translations/de.ts [hunk 1]
```diff
@@ -102,50 +102,54 @@ Zu hohe Auflösungen können daher zu Problemen in Videoanrufen führen.</transl
         <translation>qTox lädt Update %1 herunter</translation>
     </message>
     <message>
-        <source>A list of all known issues may be found at our %1 at Github. If you discover a bug or security vulnerability within qTox, please %3 according to the guidelines in our %2 wiki article.</source>
-        <translation>Eine Liste aller bekannten Probleme findest du in unserem %1 auf GitHub. Wenn du einen Fehler oder eine Sicherheitslücke in qTox findest, %3 bitte wie in unserem Wiki-Artikel %2 beschrieben.</translation>
+        <source>Original author: %1</source>
+        <translation>Originalauthor</translation>
```
Comment (Diadlo) [inline translations/de.ts:106]: Where is `%1`?

File: translations/de.ts [hunk 2]
```diff
@@ -102,50 +102,54 @@ Zu hohe Auflösungen können daher zu Problemen in Videoanrufen führen.</transl
...
+        <source>bug-tracker</source>
+        <comment>Replaces `%1` in the `A list of all knownâ¦`</comment>
+        <translation type="unfinished">Bug-Tracker (Englisch)</translation>
     </message>
     <message>
-        <source>You are using qTox version %1.</source>
-        <translation>Du verwendest qTox Version %1.</translation>
+        <source>Writing Useful Bug Reports</source>
+        <comment>Replaces `%2` in the `A list of all knownâ¦`</comment>
+        <translation type="unfinished">„Writing Useful Bug Reports“ (Englisch)</translation>
```
Comment (Diadlo) [inline translations/de.ts:138]: Seems wrong

File: translations/et.ts [hunk 3]
```diff
@@ -104,48 +104,54 @@ mis omakorda võib tekitada videokõnede pidamisel probleeme.</translation>
...
+        <source>bug-tracker</source>
+        <comment>Replaces `%1` in the `A list of all knownâ¦`</comment>
+        <translation type="unfinished">veahalduse süsteemis (inglise keeles)</translation>
     </message>
     <message>
-        <source>You are using qTox version %1.</source>
-        <translation>Kasutad qToxi versiooni %1.</translation>
+        <source>Writing Useful Bug Reports</source>
+        <comment>Replaces `%2` in the `A list of all knownâ¦`</comment>
+        <translation type="unfinished">„Writing Useful Bug Reports“ (inglise keeles)</translation>
```
Comment (Diadlo) [inline translations/et.ts:140]: Seems wrong too

**Pull request merged**